tests/Component/Basket/BasketTest.php

```diff
@@ -413,6 +413,24 @@ public function testClean()
         $this->assertEquals(1, count($basket->getBasketElements()));
     }
 
+    public function testRemoveElementWithNotExistingProduct()
+    {
+        $basket = $this->getPreparedBasket();
+        $product = $this->getMockProduct();
+
+        $basketElement = $this->createMock('Sonata\Component\Basket\BasketElementInterface');
```

Comment:
Please use `::class` constant wherever possible ❗️ 

Thank you 🙏

Reply:
Yep, forgot it. We need a PR to replace these strings in all test files.

Reply:
> We need a PR to replace these strings in all test files.

should be done through the whole code base and all sonata bundles 😄